src/main/java/com/google/sps/servlets/AccountData.java

```diff
@@ -0,0 +1,118 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import com.google.gson.Gson;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.*;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+
+/** Servlet that returns all account related info (links to login pages/account info). */
+@WebServlet("/account")
+public class AccountData extends HttpServlet {
+
+  public final class JsonHelper{
```

Comment:
For the starter project this was in a separate file.  Might be cleaner to move to another file. Also I would suggest a more descriptive name potentially (account, user, ...)? And probably should include description of class.

Reply:
Are you referring to the JsonHelper class? I feel like that would be easier in the same file, since we only use the contents of that class in this file (since its a helper class).

Reply:
I think it is fine to maintain it in the same file as discussed at the stand up but I would still suggest changing the name.

Reply:
changed name to JsonAccountInfo